app/assets/javascripts/components/common/campaign_navbar.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+// import CampaignOresPlot from '../campaign/campaign_ores_plot.jsx';
+import { NavLink } from 'react-router-dom';
+// import Alert from '../alerts/alert.jsx';
+
+const CampaignNavbar = ({ campaign }) => {
+  return (
+    <div className="campaign-nav__wrapper">
+      <div className="campaign_navigation">
+        <div className="container">
+          <a className="nav__item">
+            <h2 className="title">Campaign: {campaign.title}</h2>
+          </a>
+          <nav>
+            <div className="nav__item" id="overview-link">
+              <p><NavLink activeClassName="active" to={`/campaigns/${campaign.slug}/overview`}>Home</NavLink></p>
```

Comment:
This one shold be an anchor tag for now; we'll just use NavLink for the tabs that are being converted.

Reply:
I changed the Home tab, NavLink to the anchor tag.